context/app/static/js/shared-styles/alerts/index.js

```diff
@@ -26,6 +26,11 @@ const StyledAlert = styled(OutlinedAlert)`
     color: ${(props) => props.theme.palette.text.primary};
   }
   margin-bottom: ${(props) => props.$marginBottom || 0}px;
+
+  > div {
+    display: flex;
+    align-items: center;
+  }
```

Comment:
What necessitated this? We should style the instance of `Alert` being used, not the shared component.

Reply:
This is for the alignment
<img width="830" alt="image" src="https://github.com/hubmapconsortium/portal-ui/assets/45410262/74ea25ef-2bee-4464-82a0-a0b9d552d04e">

The alignment was off for all of the shared styles so this will fix all instances.